core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -227,7 +227,8 @@ require(
 require(
   'pages/exploration-editor-page/services/exploration-param-specs.service.ts');
 require(
-  'pages/exploration-editor-page/services/exploration-image-counter.service.ts');
+  'pages/exploration-editor-page/services/exploration-image-counter.' +
```

Comment:
If you don't need it, don't import it.

Reply:
i need it.
I have used it, you can see line no. 386